vital/exceptions/math.cxx

```diff
@@ -51,14 +51,14 @@ math_exception
 }
 
 
-non_invertible_matrix
-::non_invertible_matrix() noexcept
+non_invertible_transform
```

Comment:
I would add this as a new exception, not rename the previous.  Or maybe you can add a new common base class called `non_invertible` from which both `non_invertible_matrix` and `non_invertible_transform` derive.

In general, a matrix is not always a transform and a transform is not always a matrix.  So we may want to keep these concepts separate.

Reply:
I considered that, however, the only (prior) use of this exception was `homography::inverse`. Since `homography::inverse` implements `transform::inverse`, it should throw the same exception (or a subclass thereof) as any other implementation of `transform::inverse`.

Reply:
Is there a reason to handle non-invertable matrices different from non-invertable homographies? If not, then one exception should be sufficient. (i.e. `non_invertable`)